rules/aip0133/request_required_fields.go

```diff
@@ -45,8 +45,9 @@ var requestRequiredFields = &lint.MethodRule{
 			if !utils.GetFieldBehavior(f).Contains("REQUIRED") {
 				continue
 			}
-			// Skip the check with the field that is the body.
-			if t := f.GetMessageType(); t != nil && t.GetName() == resourceMsgName {
+			// Skip the check with the field that is the resource, which for
+			// Standard Create, is the output type.
+			if t := f.GetMessageType(); t != nil && t.GetName() == ot.GetName() {
```

Comment:
FYI 134/request_required_fields has the same issue.

And I think there may be a slightly more robust way to do this -> check the message type rather than the name. This is pretty easily done by omitting an input field that has the same type as the output field.  

It gets rid of the verifying that the name field matches the resource singular, but that's done in a separate linter rule anyway.

Reply:
I *think* that's what this check does already. It compares the request field's message type name to the name of the output type (which should be the resource for Standard Create).

Am I misunderstanding your comment?

Reply:
I'll do 0134 in a separate PR if you don't mind

Reply:
I was suggesting direct pointer checks vs name checks (to avoid the same message). But in practice yes I think it'll work fine.

Reply:
I'm fine with it in a separate PR. thanks!

Reply:
> I was suggesting direct pointer checks vs name checks (to avoid the same message). But in practice yes I think it'll work fine.

Oh yeah I see...yeah Idk I'm scared by that haha